lib/util/git/types.ts

```diff
@@ -17,6 +17,8 @@ export interface StorageConfig {
   extraCloneOpts?: GitOptions;
   cloneSubmodules?: boolean;
   fullClone?: boolean;
+  token?: string;
+  platformGitCredentialsFile?: boolean;
```

Comment:
instead of passing a boolean, pass the content. pass null / undefined when not used.